components/proxy/src/main/java/com/hotels/styx/routing/config/Builtins.java

```diff
@@ -220,4 +231,24 @@ public static InetServer buildServer(
 
         return constructor.create(name, context, serverDef.config(), serverDb);
     }
+
+    /**
+     * Builds a Styx server.
```

Comment:
Should this javadoc refer to Netty Executors rather than Styx Servers?

Reply:
👍